i18n/de/docusaurus-plugin-content-docs/current/help-center/using-famedly/chats.mdx

```diff
@@ -194,6 +194,18 @@ Das Laden des Archivs kann einige Zeit in Anspruch nehmen.
   </TabItem>
 </Tabs>
 
+## Den Online-Status anderer Nutzer prüfen
+
+Sie können den Online-Status anderer Nutzer auf den Avataren in der Raumliste, der Kontaktliste (beim Erstellen eines neuen Chats), in der Titelleiste von 1:1 Chats sowie in Gruppenchats in den einzelnen Nachrichten und der Mitgliederliste sehen.  
```

Comment:
This I do not understand. How can I see the online status in the single messages within a group chat?

Reply:
In group chats every message other parties write have their avatar showed above. The avatar has a presence indicator: 
![image](https://github.com/famedly/helpcenter/assets/139118252/41eb95d1-5a14-4406-979b-888499a0dbb7)